portfolio.html

```diff
@@ -54,5 +54,113 @@ <h2>I am a Software Developer</h2>
         </div>
       </div>
     </section>
+
+    <section class="works">
+      <div id="set1">
+        <h1>My Recent Works</h1>
+      </div>
+      <div>
+        <hr class="solid" />
+      </div>
+      <div>
+        <img
+          class="set-image"
+          src="/images/ImgPlaceholder.png"
+          alt="works image"
+        />
+      </div>
+      <div id="set2">
+        <h2>Multi-Post Stories</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+      </div>
+      <div id="lang">
+        <img src="/images/Icons/languages/css.png" alt="csslang" />
+        <img src="/images/Icons/languages/html.png" alt="htmllang" />
+        <img src="/images/Icons/languages/bootstrap.png" alt="bootlang" />
+        <img src="/images/Icons/languages/ruby.png" alt="rubylang" />
+      </div>
+      <div class="project-button">
+        <a href="projectlink"
+          ><img
+            src="/images/Icons/Button/Enable.png"
+            alt="button
+        icon"
+        /></a>
+      </div>
+    </section>
+    <section class="project-works">
+      <div class="smallworks">
+        <h1>Professional Art Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <div>
+          <img
+            src="/images/Icons/languages/darkicons/html.png"
+            alt="htmllang"
+          />
+          <img
+            src="/images/Icons/languages/darkicons/bootstrap.png"
+            alt="bootlang"
+          />
+          <img
+            src="/images/Icons/languages/darkicons/ruby.png"
+            alt="rubylang"
+          />
+        </div>
```

Comment:
Your works section doesn't match the Figma design, you should try to stick to the design as possible, please fix this : 👍😃

![image](https://user-images.githubusercontent.com/87448628/159058160-5b75d274-1e5d-48c0-b7b5-3705e2155265.png)

Figma Design : 
![image](https://user-images.githubusercontent.com/87448628/159058329-25faa0c4-0ed4-4be6-ae58-4cc0fd571e96.png)